test-cases/longevity/longevity-nosqlbench-3h.yaml

```diff
@@ -15,6 +15,7 @@ gce_instance_type_loader: 'e2-standard-4'
 ssh_transport: 'libssh2'
 
 nemesis_class_name: 'SisyphusMonkey'
+nemesis_include_filter: ['limited']
```

Comment:
It's not limited, it should be non-disruptive.
But now I see that i don't have way to limit with "non-disruptive".
You can change this one back to ChaosMonkey for now and set to non-disruptive.